system/lib/wasmfs/syscalls.cpp

```diff
@@ -1112,6 +1112,46 @@ int __syscall_ftruncate64(int fd, uint64_t size) {
   return ret;
 }
 
+static bool isTTY(int fd) {
+  // TODO: Full TTY support. For now, just see stdin/out/err as terminals and
+  //       nothing else.
+  return fd == STDIN_FILENO || fd == STDOUT_FILENO || fd == STDERR_FILENO;
+}
```

Comment:
Can we at least check for `File` equality with `/dev/stdin`, etc? That will be more robust against the standard streams being moved or reassigned, etc. Even better would be to add a virtual method to `File` and have our stream implementations override it.

Reply:
Good idea to check for equality on the singleton objects, done.

We could add a virtual method to File, but I'm not sure what it should look like - "isTTY"? Or should it be more general and represent all types of special devices? As our earliest adopters of WasmFS won't be using TTY support, I think it's best to defer work on that.

Reply:
Yeah I was imagining `isTTY`, but passing on that for now seems fine.